docs/content/posts/2024-06-14-release-1.19.1.md

```diff
@@ -0,0 +1,128 @@
+---
+title:  "Apache Flink 1.19.1 Release Announcement"
+date: "2024-06-14T00:00:00.000Z"
+aliases:
+- /news/2024/06/14/release-1.19.1.html
```

Comment:
tips: please update the date when fire the release.

Reply:
Released artifacts on 06-14 so I will keep this date!